### PR TITLE
Visual tests for amp-list and mustache

### PR DESCRIPTION
**Pull request by dvoytenko**
Partial for #11417.

### CODE REVIEW

**Comment by dvoytenko** [comment]
@rsimha-amp I assume I should approve percy right before merge (assuming approval), right?

File: test/visual-diff/visual-tests.json [hunk 1]
```diff
@@ -70,6 +70,16 @@
       "url": "examples/visual-tests/amp-by-example/amp-by-example.html",
       "name": "Amp By Example"
     },
+
+    {
+      "url": "examples/visual-tests/amp-list/amp-list.amp.html",
+      "name": "AMP List and Mustache",
+      "loading_complete_css": [
+        "#list1",
```
Comment (rsimha-amp) [inline test/visual-diff/visual-tests.json]: Curious: Does this actually end up waiting? I thought css classes begin with a period, like `.list1`.

Comment (rsimha-amp) [reply test/visual-diff/visual-tests.json]: I see from the HTML files that it's an `id`, not a `class`. I believe capybara needs you to use a class name here for it to wait until the element appears. If not, there might be flakiness in this test.

Comment (dvoytenko) [reply test/visual-diff/visual-tests.json]: All other appear to be CSS selectors. So I assume it will lookup ID CSS selector just as well?

Comment (dvoytenko) [reply test/visual-diff/visual-tests.json]: I changed to classes to avoid flakiness, but I'd still like to know if ID-selectors supported.

**Comment by rsimha-amp** [comment]
@dvoytenko Thanks for merging this PR

From looking at https://percy.io/ampproject/amphtml/builds/359123?snapshot=13181241, I wonder if the content can be made to span the entire page width when it's displayed in the desktop resolution. Might be worth considering as you write more visual tests.

**Pull request merged**